indico_zodbimport/modules/event_papers.py

```diff
@@ -169,6 +223,87 @@ def _migrate_paper_roles(self, contribution):
             elif role.role == PaperReviewingRoleType.editor:
                 contribution.paper_layout_reviewers.add(role.user)
 
+    def _migrate_review(self, contribution, old_judgment, review_type):
+        # Consider legacy custom states the same as "to be corrected"
+        proposed_action = JUDGMENT_STATE_PAPER_ACTION_MAP.get(int(old_judgment._judgement._id),
+                                                              PaperAction.to_be_corrected)
+        review = PaperReview(user=old_judgment._author.user, comment=old_judgment._comments, type=review_type,
```

Comment:
`convert_to_unicode()` all the string values; this also takes care of stripping and a removing garbage (control chars).
Probably it's needed in a few other places too